data/hai/hai news.txt

```diff
@@ -0,0 +1,251 @@
+# Copyright (c) 2020 by Nomadic Volcano
+# Based on earlier text copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# Visual suggestion: a bit quirky or weird; expression is content, confident, or stoic
+news "food in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Visiting chef"
+			"Food connoisseur"
+			"Tourist"
+			"Tour guide"
+	portrait
+		portrait/human26
+		portrait/human31
+		portrait/human57
+		portrait/human68
+	message
+		word
```

Comment:
Might want to add a message about the pickled acorns mentioned in the Centipede's description.

Reply:
"Back home, my favorite snack was pickled walnuts. Here, the pickled acorns are amazing!"